### PR TITLE
feat: 🎸 display credentials for static credential store

### PR DESCRIPTION
**Pull request by DhariniJeeva**
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/5177)

This pr lists all the available credentials within a static credential store and adds support to create a new credential

tested against boundary instance:  
Screen Shot 2022-07-20 at 3 36 22 PM

admin ui- https://boundary-4at9iyc7v-hashicorp.vercel.app/scopes/s_1sheljmjgm/credential-stores/cs_20nu3mgl4y/credentials

create a new credential: 
Screen Shot 2022-07-22 at 11 25 59 AM

### CODE REVIEW

File: addons/core/translations/resources/en-us.yaml [hunk 1]
```diff
@@ -378,6 +378,20 @@ credential-library:
...
+  messages:
+    none:
+      title: No Credentials Available
+      description: No Credentials available in this credential store.
+  types:
+    username_password: Username & Password
+  help:
+    username_password: This host requires a username and password to connect.
+    username: The username to use when signing in with this credential store.
+    password: The password to use when signing in with this credential store.
```
Comment (cameronperera) [inline addons/core/translations/resources/en-us.yaml:394]: Talking with Joshua to determine if this is the test we want to use for username and password. Not a blocking issue, just wanted you to be aware.

**Review by cameronperera — APPROVED** [review]
LGTM!
Left a couple of comments but not blocking.

File: ui/admin/app/components/form/credential/index.hbs [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
```
[the remark this answers is not in the document]
Comment (DhariniJeeva) [reply ui/admin/app/components/form/credential/index.hbs:5]: no particular reason. we follow both the styles in our codebase.

**Review by lisbet-alvarez — APPROVED** [review]

**Review by lisbet-alvarez — COMMENTED** [review]
Wow! A lot of amazing work 🤩

**Pull request merged**